src/services/MoxfieldService.ts

```diff
@@ -0,0 +1,47 @@
+import axios from "axios";
+import { useCallback } from "react";
+
+const useGetMoxfieldProfile = () => {
+    const endpoint = "https://chatterfang.onrender.com/moxfield/";
+
+    return useCallback(async (moxfieldId: string) => {
```

Comment:
for the sake of good practice, can we add a return type to the function that is being returned by the useCallback?